apps/ledger-live-desktop/src/main/window-lifecycle.ts

```diff
@@ -102,7 +108,7 @@ export async function createMainWindow({ dimensions, positions }: any, settings:
   if (DEV_TOOLS && !process.env.DISABLE_DEV_TOOLS) {
     mainWindow.webContents.on("did-frame-finish-load", () => {
       if (mainWindow) {
-        mainWindow.webContents.once("devtools-open", () => {
+        mainWindow.webContents.once("devtools-opened", () => {
```

Comment:
does it mean we had a bug?

Reply:
I believe so yes :(

https://www.electronjs.org/docs/latest/api/web-contents#event-devtools-opened